scipy/spatial/kdtree.py

```diff
@@ -222,6 +222,9 @@ class KDTree(cKDTree):
         is the boxsize along i-th dimension. The input data shall be wrapped
         into :math:`[0, L_i)`. A ValueError is raised if any of the data is
         outside of this bound.
+    build_workers : int, optional
+        Number of workers to use for parallel processing. If <=1 one thread is
+        used. Default: 1.
```

Comment:
```suggestion
    workers : int, optional
        Number of workers to use for parallel processing. If -1 is given all
        CPU threads are used. Default is 1.
```

Reply:
missed this one. I will push it with some later changes.

Reply:
Amended to https://github.com/scipy/scipy/pull/14481/commits/5e74b0a3f67801bd7f67550eed4e397fc4fbb351 .